ocs_ci/ocs/acm/acm.py

```diff
@@ -619,3 +624,301 @@ def import_clusters_with_acm():
         )
     else:
         import_clusters_via_cli(clusters)
+
+
+class CreateClusterViaACM(object):
+    def __init__(self):
+        self.cluster_name = config.ACM_CONFIG["cluster_name"]
+
+        ssh_priv_key_path = os.path.expanduser(config.DEPLOYMENT["ssh_key_private"])
+        with open(ssh_priv_key_path, "r") as fp:
+            self.ssh_priv_key = fp.read()
+
+        ssh_public_key_path = os.path.expanduser(config.DEPLOYMENT["ssh_key"])
+        with open(ssh_public_key_path, "r") as fp:
+            self.ssh_public_key = fp.read()
+
+        with open(constants.VSPHERE_CA_FILE_PATH, "r") as fp:
+            self.vsphere_ca = fp.read()
+
+        with open("/home/oviner/OCS-AP/ocs-ci/data/pull-secret", "r") as fp:
+            self.pull_secret = fp.read()
+
+        self.ips = ["1", "2"]
+
+        self.secret_yaml = templating.load_yaml(constants.ACM_SECRET_YAML)
+
+    def create_credential(self):
+        """
+
+        Returns:
+
+        """
+        cred_vsphere = templating.load_yaml(constants.ACM_VSPHERE_CRED_YAML)
+        cred_vsphere["stringData"]["vCenter"] = config.ENV_DATA["vsphere_server"]
```

Comment:
```suggestion
        cred_vsphere["stringData"]["vCenter"] = config[cluster_index].ENV_DATA["vsphere_server"]
```

We will need have some construction to read the config data from specific cluster as we discussed on meeting